### DISCUSSION TITLE
Is Hylang ready for production use?

### DISCUSSION THREAD

**Discussion by mod0** [hylang/hy#2311] [Q&A] [answered]
Can I mix hylang and python in a production environment? Wonder what the core team thinks about stability and performance.

**Comment by Kodiologist** [comment, marked as answer]
Performance is essentially the same as Python (except maybe in extreme cases that we don't test involving parsing a huge amount of code). The known bugs are generally not so bad (see the issues list). However, the language is still changing. You can see from NEWS that with each release comes breaking changes. I use Hy for real work and so do several other core maintainers, but until 1.0 is out, you should expect to need to change your code to follow language changes, unless you're happy with staying on one release for a long time.

**Reply by mod0** [reply]
Thank you! Looking forward to 1.0! I will look at the other resources you've pointed me to.
